javascript/javascript1/week2/studentmanager.js

```diff
@@ -0,0 +1,38 @@
+const class07Students = [];
+const queen='Margrethe II';
+function addStudentToClass(studentName) {
+  if (!class07Students.includes(queen)){
+      class07Students.push(queen);
+      console.log(`queen ${queen} has been added to the the class07`);
+} if  (class07Students.length>6){
+    console.log("Cannot add more students to class 07");
+      } else if (class07Students.length){
+      console.log(`student ${studentName} is included in the class07`)
+  }if (studentName===''){
+    console.log("you cannot add an empty string to the class")
```

Comment:
I would start with this since you don't want to add students without a proper name to the class. You should also check if the name is a string at all.
Test with `addStudentToClass("");` `addStudentToClass(true);`